Makefile

```diff
@@ -24,6 +24,7 @@ BUILD_DATE = $(shell date -u)
 BUILD_HASH = $(shell git rev-parse HEAD)
 # If we don't set the build number it defaults to dev
 ifeq ($(BUILD_NUMBER),)
+	BUILD_DATE := dev
```

Comment:
0/5, I think `n/a` feels better. `dev` does not feel like a date. :P

Reply:
Clearly not a blocker but do we need to define this in the first place? Could it just be left blank in this case?

Reply:
I think it is not needed, but it is interesting to have it exposed like "Hey, there is this thing, it doesn't apply on develoment environment, but it is there".

Reply:
Fair enough :+1: